contracts/BirdSwap.sol

```diff
@@ -49,42 +49,43 @@ contract BirdSwap is IBirdSwap, UUPSUpgradeable, ReentrancyGuardUpgradeable, IER
         uint256 _askPrice,
         uint256 _royaltyFeeBps
     ) external onlyTokenSeller(_tokenId) nonReentrant {
+        require(_royaltyFeeBps <= 1000, "createAsk royalty fee basis points must be less than or equal to 10%");
+
         address tokenOwner = moonbirds.ownerOf(_tokenId);
         // If the Moonbird is already escrowed in BirdSwap, the ownerOf will return this contract's address.
         if (tokenOwner == address(this)) {
             tokenOwner = moonbirdTransferredFromOwner[_tokenId];
         }
 
-        require(_royaltyFeeBps <= 10000, "createAsk royalty fee basis points must be less than or equal to 10000");
-
-        // prevent multiple asks from being created for the same token ID
-        if (askForMoonbird[_tokenId].seller != address(0)) {
-            _cancelAsk(_tokenId);
-        }
 
```

Comment:
If we remove this line, then the CancelAsk event is no longer emitted, which means that that it may make it tricky to filter out showing the previous Ask. Do you think this will be an issue?

Reply:
I see, I can probably just emit an event if there's a new listing from a new seller for the same tokenId.